### DISCUSSION TITLE
Table 10, Panel 3

### DISCUSSION THREAD

**Discussion by stefanhenneking** [cssi-pi-community/2022-meeting#79] [Panel 3 roundtable (Tuesday morning)]
Table 10, Panel 3 discussion

**Comment by stefanhenneking** [comment]
To promote sustainability, panelist M. Zenter (SDSC) suggested rewarding "non-reinvention." 
Any ideas how to implement such a reward system?

**Comment by stefanhenneking** [comment]
How can NSF be involved in data curation? Can NSF request funding specifically for providing data storage, curation?
@tjhacker

**Comment by bhazelton** [comment]
# Creating a service for data archinving & curation
- provide data curation + archive as a service (large facility)
- require NSF grantees to deposit data there as appropriate.
- promulgate best practices

# Software sustainability vs innovation
- software can become a platform -- essential for projects that built on top of them. Not fair to expect the innovators to own them, also not good/fair for the people who depend on them because they need to be stable. Someone needs to make it good and stable but that costs money.
- Should maintenance really be only done by non-innovators as was argued by the panel? We feel that many software infrastructures should be maintained by scientists/innovators with part time (e.g. 20%) support because they are not large projects, so this may be enough, and because they require some domain knowledge.
- Some kind of supplement structure that doesn't require constant growth or innovation. This should be for small enhancements, bug fixes and maintenance for projects that have significant community dependence. This may have a larger impact than new innovation because many people already depend on it. A supplement could fund either the team on the proposal or a different team that supports software that the proposal depends on.
